ckanext/dcat/utils.py

```diff
@@ -1,7 +1,16 @@
 import logging
 import uuid
+import json
 
-from pylons import config
+from ckantoolkit import config, h
+
+try:
+    # CKAN >= 2.6
+    from ckan.exceptions import HelperError
+except ImportError:
+    # CKAN < 2.6
+    class HelperError(Exception):
+        pass
```

Comment:
I already added the `HelperError` to ckantoolkit to avoid this "hack" (https://github.com/ckan/ckantoolkit/pull/6)